### PR TITLE
Update FileCap Server and Added FileCap plugins

### PR DESCRIPTION
**Pull request by rmu1987**
Fixed in FileCap Server version 5.1.1.
Added FileCap Plugins, not vulnerable.

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -805,7 +805,8 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 |:----------------|:----------------|:--------------------:|:---------------:|:----------------|-----------:|
 |F5| All products | |Not Vuln | F5 products themselves are not vulnerable, but F5 published guidance on mitigating through BIG-IP ASM/Advanced WAF and NGINX App Protect|[source](https://support.f5.com/csp/article/K19026212)|
 | Fiix | CMMS core | V5  | Fix | | [source](https://rockwellautomation.custhelp.com/app/answers/answer_view/a_id/1133605) |
-|FileCap| All products | <5.1.0 | Vulnerable | Fix: 5.1.1 |[source](https://mailchi.mp/3f82266e0717/filecap-update-version-511)|
+|FileCap| FileCap Server | 5.1.1 | Fix | |[source](https://mailchi.mp/3f82266e0717/filecap-update-version-511)|
+|FileCap| FileCap Plugins || Not vuln | ||
```
Comment (maertsen) [inline software/README.md:809]: Can you add a source for this this line? I don't see anything about plugins stated.

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @rmu1987, thanks for your contribution. Can you review my comments?

**Comment by jspaans91** [comment]
@rmu1987 maybe you can link to https://filecap.com/blog-posts/detecteer-log4shell-exploitatie-pogingen, version 5.1.1 is still vulnarable. The vulnarability is completely fixed in version 5.1.2.

Also set the version to <= 5.1.1

**Comment by rmu1987** [comment]
@jspaans91 FileCap Server version 5.1.1 fixed CVE-2021-44228 and FileCap Server 5.1.2 fixed CVE-2021-45046.

**Comment by maertsen** [comment]
Please add a commit to make the requested change; we have too many incoming PRs to manually edit each one. Thanks for your help!

File: software/README.md (hunk 1 above)
Comment (rmu1987) [reply software/README.md]: @maertsen i added a source for the FileCap plugin.

**Comment by maertsen** [comment]
thanks!

**Pull request merged**